lib/Backend/GlobOpt.cpp

```diff
@@ -527,6 +532,9 @@ GlobOpt::ForwardPass()
     // Make sure we free most of them.
     Assert(freedCount >= spilledCount);
 
+    JitAdelete(alloc, this->changedSymsAfterIncBailoutCandidate);
```

Comment:
Will this-alloc be freed in END_CODEGEN_PHASE, I don't see it is freed in it.

---
In reply to: [72335803](https://github.com/Microsoft/ChakraCore/pull/1337#discussion_r72335803) [](ancestors = 72335803)